src/main/kotlin/com/github/vol0n/utbotcppclion/services/Client.kt

```diff
@@ -0,0 +1,271 @@
+package com.github.vol0n.utbotcppclion.services
+
+import com.github.vol0n.utbotcppclion.actions.AskServerToGenerateBuildDir
+import com.github.vol0n.utbotcppclion.actions.AskServerToGenerateJson
+import com.github.vol0n.utbotcppclion.actions.getProjectConfigRequestMessage
+import com.github.vol0n.utbotcppclion.actions.utils.notifyError
+import com.github.vol0n.utbotcppclion.actions.utils.notifyInfo
+import com.github.vol0n.utbotcppclion.actions.utils.notifyUnknownResponse
+import com.github.vol0n.utbotcppclion.client.GrpcStarter
+import com.github.vol0n.utbotcppclion.messaging.ConnectionStatus
+import com.github.vol0n.utbotcppclion.messaging.UTBotConnectionChangedNotifier
+import com.github.vol0n.utbotcppclion.ui.UTBotRequestProgressIndicator
+import com.github.vol0n.utbotcppclion.utils.createFileAndMakeDirs
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
+import io.grpc.Metadata
+import io.grpc.Metadata.ASCII_STRING_MARSHALLER
+import io.grpc.stub.MetadataUtils
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.swing.Swing
+import kotlinx.coroutines.withContext
+import kotlinx.coroutines.withTimeout
+import testsgen.Testgen
```

Comment:
Please move testgen imports near to the imports of the project as they are related. Use some import structure and more empty lines, for example:
```
UTBotCppClion imports
Empty line
Generated grpc imports like testgen
Empty line
Intellij imports
grpc imports
logger imports
Empty line
Kotlin imports
Java imports
```

And use this structure for other files as well (Maybe it can be configured in Intellij Idea)